vnext/Shared/InspectorPackagerConnection.cpp

```diff
@@ -0,0 +1,176 @@
+
+#include "pch.h"
+
+#include <folly/json.h>
+#include "InspectorPackagerConnection.h"
+
+namespace Microsoft::ReactNative {
+
+namespace {
+
+struct InspectorProtocol {
+  static constexpr const char InspectorMessage_EVENT[] = "event";
```

Comment:
The naming here is a bit different than I usually see.

What do you think about using a namespace instead of a prefix in the variable name?
```cpp
namespace InspectorMessage {
static constexpr const char Event = "event";
}
```

Reply:
I'm removing the repeater usage of "Inspector" from names.. It would've been nice to namespace these as you suggested, but looks like we can't have namespace inside struct !